frontend/src/components/header/Header.js

```diff
@@ -1,34 +1,31 @@
 import React from 'react';
 import { BrowserRouter as Router, Link } from 'react-router-dom';
 import './Header.css';
+import HeaderTitle from './HeaderTitle';
+
+const loggedinHeaderItems = [
+  { link: 'Settings', route: '/settings' },
+  { link: 'Logout', route: '/logout' },
+];
+const loggedoutHeaderItems = [
+  { link: 'Login', route: '/login' },
+  { link: 'Register', route: '/register' },
+];
 
 export default function ({ kingdomName }) {
-  let title = <h1 className="header">{'Tribes of Vulpes'}</h1>;
-  const loggedinHeaderItems = [
-    { link: 'Settings', route: '/settings' },
-    { link: 'Logout', route: '/logout' },
-  ];
-  const loggedoutHeaderItems = [
-    { link: 'Login', route: '/login' },
-    { link: 'Register', route: '/register' },
-  ];
   let headerItems = loggedoutHeaderItems;
-
   const isToken = localStorage.getItem('token') ? true : false;
 
   if (isToken && kingdomName) {
-    title = (
-      <Link className="header" to={'/buildings'}>
-        <h1 className="header">{kingdomName}</h1>
-      </Link>
-    );
     headerItems = loggedinHeaderItems;
   }
   return (
     <Router>
       <nav className="header">
-        {title}
-
+        <HeaderTitle
+          name={isToken && kingdomName ? kingdomName : undefined}
```

Comment:
@kkeszthe actually this can be simply
```suggestion
          name={isToken && kingdomName && kingdomName}
```